canonical-webteam/global-nav
# Fixes for integration into ubuntu.com

## Done
Some styling to override the reset used on ubuntu.com.

## QA
- Pull branch
- Install if not run before with `yarn install`
- Run `gulp build`
- Open the index.html file
- Check the nav looks as it did before

## Details
Fixes https://github.com/canonical-webteam/global-nav/issues/29
